x-pack/plugin/kql/src/test/resources/supported-queries

```diff
@@ -67,6 +67,8 @@ foo_field:foo OR foo_field:foo bar AND foo bar
 foo_field:foo AND (foo_field:foo bar OR foo bar)
 foo_field:foo AND (foo_field:foo bar OR foo bar)
 foo_field:foo OR (foo_field:foo bar OR foo bar)
+foo AND
```

Comment:
ℹ️ KQL support queries with a trailing AND operator and just ignore it.

The grammar has been updated to support it and these queries have been moved from `unsupported-queries` to `supported-queries`